### DISCUSSION TITLE
Welcome to developer-portal-starter Discussions!

### DISCUSSION THREAD

**Discussion by adamaltman** [Redocly/developer-portal-starter#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JeroenVanHautte** [comment]
Hi Adam, thanks for starting this thread! At TechWolf, we've been very happy with redocly - it's been saving us a lot of time while keeping our docs clean. We're currently looking into a transition of our versioning towards a date-based versioning model, along with indicating which features are experimental in our documentation. Is that something that would be possible today or in the near future with redocly & the dev portal?

**Reply by adamaltman** [reply]
Is the date-based versioning model like Stripe? Will users of the API be able to specify the version (kind of like you can do with Stripe) -- Stripe gives the option of the most recent version or the version your API key was used one.

**Reply by JeroenVanHautte** [reply]
> Is the date-based versioning model like Stripe? Will users of the API be able to specify the version (kind of like you can do with Stripe) -- Stripe gives the option of the most recent version or the version your API key was used one.

Yes, it would be exactly like stripe!

**Reply by adamaltman** [reply]
If I recall, Stripe shows the latest version of the docs unless you are logged in and your API key is pinned to a previous version. In that case, Stripe shows that older version which has some message that says "would you like to see the docs for the latest API version?" (or something like that...)

**Reply by JeroenVanHautte** [reply]
Yes - however, initially we'd just like to be able to have multiple versions available for different endpoints. Is that in some way within the scope of the developer portal or reference docs?

**Reply by adamaltman** [reply]
Are you planning on exposing a "select" menu (or something similar)?
